Lib/asyncio/streams.py

```diff
@@ -1366,7 +1371,7 @@ def writelines(self, data):
     def _fast_drain(self):
         # The helper tries to use fast-path to return already existing
         # complete future object if underlying transport is not paused
-        #and actual waiting for writing resume is not needed
+        # and actual waiting for writing resume is not needed
```

Comment:
I think it's ok to fix this unrelated comment spacing here. Don't worth a separate PR